core_java/twitter/src/main/java/ca/jrvs/apps/twitter/dao/TwitterDao.java

```diff
@@ -0,0 +1,136 @@
+package ca.jrvs.apps.twitter.dao;
+
+import ca.jrvs.apps.twitter.dao.helper.HttpHelper;
+import ca.jrvs.apps.twitter.dao.helper.TwitterHttpHelper;
+import ca.jrvs.apps.twitter.model.GeoLoc;
+import ca.jrvs.apps.twitter.model.Tweet;
+import ca.jrvs.apps.twitter.utils.JsonUtil;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import org.apache.http.HttpResponse;
+import org.apache.http.util.EntityUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+@org.springframework.stereotype.Repository
+public class TwitterDao implements CrdDao<Tweet, Long> {
+
+  private static final String BASE_URL = "https://api.twitter.com/1.1/";
+  private static final String POST_URL = BASE_URL + "statuses/update.json";
+  private static final String SHOW_URL = BASE_URL + "statuses/show/";
+  private static final String DELETE_URL = BASE_URL + "statuses/destroy/";
+
+  private static final int HTTP_OK = 200;
+
+  private HttpHelper httpHelper;
+  private Logger daoLogger = LoggerFactory.getLogger(TwitterDao.class);
```

Comment:
use IDE to rename daoLogger to logger (IDE > right-click > refactor )..
logger is a common naming convention